support/client/lib/vwf/model/mil-sym.js

```diff
@@ -452,36 +452,29 @@ define( [ "module",
                         case "symbolType":
                         case "bounds":
                             if ( node.nodeType === "missionGfx" && propertyValue !== undefined &  propertyValue !== null ) {
-                                if ( node[ propertyName ] !== propertyValue ) {
-                                    node[ propertyName ] = propertyValue;
-                                    renderImage = basicPropertiesMet( node );
-                                }
+                                node[ propertyName ] = propertyValue;
+                                renderImage = basicPropertiesMet( node );
```

Comment:
I'm curious ... why did this need to change to set the value even if it's the same as what already existed?